lib/presentation/widget/myspace/my_space_viewmodel.dart

```diff
@@ -231,8 +232,25 @@ class MySpaceViewModel extends BaseViewModel {
     };
   }
 
+  void selectItem(Document selectedDocument) {
+    store.dispatch(MySpaceSelectDocumentAction(selectedDocument));
+  }
+
+  void toggleSelectAllDocuments() {
+    if (store.state.mySpaceState.documentList.every((element) => element.selectMode == SelectMode.ACTIVE)) {
+      store.dispatch(MySpaceUnselectAllDocumentsAction());
+    } else {
+      store.dispatch(MySpaceSelectAllDocumentsAction());
+    }
+  }
+
+  void cancelSelection() {
+    store.dispatch(MySpaceClearSelectedDocumentsAction());
+  }
+
   @override
   void onDisposed() {
+    cancelSelection();
```

Comment:
please make the GIF for the case: 
- Select some documents in MySpace
- Open Side menu -> SharedSpace
- Back to MySpace: by `Side Menu` or `Back press button on Android`

Reply:
![test_dat](https://user-images.githubusercontent.com/48354990/103771361-e3291880-5027-11eb-9545-3a0bb4886c57.gif)

Reply:
Sound good for the back press. How about:
- Select some documents in MySpace
- Open Side menu -> SharedSpace
- Open Side menu -> MySpace

Reply:
![my_space_to_shared_space_multiple_selection](https://user-images.githubusercontent.com/48354990/103877858-ed095500-50d5-11eb-8972-935bc1f7605d.gif)

Reply:
so good.